Assets/Minigames/Fight/Scripts/Entity/Entity.cs

```diff
@@ -10,21 +10,24 @@ namespace Minigames.Fight
 {
     public class Entity : MonoBehaviour
     {
-        public WeaponController WeaponController;
```

Comment:
why did we move this to the player? does only the player have a weapon controller?